src/Homepage/Homepage.js

```diff
@@ -1,13 +1,13 @@
 import React, { Component } from 'react';
 import { WelcomingSection } from '../WelcomingSection/WelcomingSection'
 import  Cards  from '../Cards/Cards'
+import fetcher from '../API/APIcalls';
 import './Homepage.scss';
 
 export class Homepage extends Component {
-    constructor(props){
-        super(props)
+    constructor(){
+        super()

```

Comment:
Is there anything we will have to pass props in as through here? I didn't think so, but now I'm second guessing myself. Maybe for user ratings from App => login?